inputstream.adaptive/addon.xml.in

```diff
@@ -13,12 +13,16 @@
     listitemprops="license_type|license_key|license_data|license_flags|manifest_type|server_certificate|stream_headers|manifest_update_parameter|original_audio_language|media_renewal_url|media_renewal_time|max_bandwidth|play_timeshift_buffer"
     library_@PLATFORM@="@LIBRARY_FILENAME@"/>
   <extension point="xbmc.addon.metadata">
-    <summary lang="en">InputStream client for adaptive streams</summary>
-	<summary lang="es">Cliente InputStream para flujo de datos adaptativos</summary>
-    <description lang="en">InputStream client for adaptive streams</description>
-	<description lang="es">Cliente InputStream para flujo de datos adaptativos</description>
+    <summary lang="en_GB">InputStream client for adaptive streams</summary>
+    <summary lang="es_ES">Cliente InputStream para flujo de datos adaptativos</summary>
+    <description lang="en_GB">InputStream client for adaptive streams</description>
+    <description lang="es_ES">Cliente InputStream para flujo de datos adaptativos</description>
     <platform>@PLATFORM@</platform>
 	<news>
+v2.5.7 (2020-05-16)
+- Update inputstream API 2.3.1
+- Update Global APi 1.2.0
```

Comment:
```suggestion
- Update Global API 1.2.0
```

Reply:
Ah yes, seen. Have them wrongly on most addons.
At home already fixed :smile:. As that after Kodi's request is, in every addon request must be triggered for build, I thought to make with them together.